src/picknik_ur_base_config/config/control/picknik_ur.ros2_control.yaml

```diff
@@ -131,6 +131,32 @@ streaming_controller:
       - position
     state_interfaces:
       - position
+      - velocity
```

Comment:
Looks like this config is a similar to `joint_trajectory_controller`. Can't you use it instead? Is the streaming controller only used by servo? In that case we could remove it completely :thinking: